prusa_slicer_post_processing_script.py

```diff
@@ -308,9 +311,10 @@ def main(gCodeFileStream,path2GCode,skipInput)->None:
                     curPrintSpeed="G1 F600"
                     messedWithSpeed=False
                     messedWithFan=False
-                    layer.prepareDeletion(featurename="Bridge",polys=layer.validpolys)
-                    if len(layer.oldpolys)>0:
-                        layer.prepareDeletion(featurename=":Solid",polys=layer.oldpolys)
+                    if gcodeWasModified:
```

Comment:
Adds a check here to make sure we succeeded in making arcs before deleting things...